api/ReadableStreamBYOBReader.json

```
@@ -221,8 +221,7 @@
             ],
             "support": {
               "chrome": {
-                "version_added": false,
-                "impl_url": "https://crbug.com/40942083"
+                "version_added": "140"
```

This actually landed in https://chromiumdash.appspot.com/commit/ad2f4eae04fe2fb53d947c8a5be0030226a01799.

```suggestion
                "version_added": "139"
```